tests/common/Configuration.cs

```diff
@@ -347,6 +347,12 @@ public static bool TryGetRootPath (out string rootPath)
 			}
 		}
 
+		public static string XamarinCatalystDll {
+			get {
+				return Path.Combine (mt_root, "lib", "mono", "Xamarin.MacCatalyst", "Xamarin.MacCatalyst.dll");
+			}
+		}
+
```

Comment:
I don't think this is needed anymore:

```suggestion
```

Reply:
Kept this because it's related to the above which is needed to GetAssembly